tools/run_tests/filter_pull_request_tests.py

```diff
@@ -108,7 +132,7 @@ def _get_changed_files(base_branch):
   """
   # git fetch might need to be called on Jenkins slave
   # todo(mattkwong): remove or uncomment below after seeing if Jenkins needs this
-  # call(['git', 'fetch'])
+  call(['git', 'fetch'])
```

Comment:
As I said before, I'm not quite sure why this would be needed - perhaps improve the comment?